static/gccdump-view.js

```diff
@@ -127,6 +127,20 @@ define(function (require) {
         this.eventHub.emit('gccDumpUIInit', this.state._compilerid);
     }
 
+    // Disable view's menu when invalid compiler has been
+    // selected after view is opened.
+    GccDump.prototype.onUiNotReady = function () {
+        this.filters.off('change');
+        this.selectize.off('change');
+
+        // disable drop down menu and buttons
+        this.selectize.disable();
+        this.domRoot.find('.dump-filters').find('.btn')
```

Comment:
Isn't `this.domRoot.find('.dump-filters .btn').prop("disabled", true);` similar (And maybe it performs better?)

Reply:
You're right. I think I ended up with this after copy/pasting/modifying some code. Fixed here and 2 other occurrences. Thanks for the review !